packages/curve-ui-kit/src/features/market-position-details/HealthDetails.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box, Typography, type Theme, useTheme } from '@mui/material'
+import type { Health } from '@ui-kit/features/market-position-details/BorrowPositionDetails'
+import { HealthBar } from '@ui-kit/features/market-position-details/HealthBar'
+import { t } from '@ui-kit/lib/i18n'
+import { Metric } from '@ui-kit/shared/ui/Metric'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+
+const { Spacing } = SizesAndSpaces
+
+const getHealthValueColor = (value: number, theme: Theme) => {
+  if (value < 5) return 'error'
+  if (value < 15) return 'warning'
+  if (value < 50) return theme.design.Color.Secondary[500]
+  if (value >= 100) return theme.design.Color.Secondary[600]
+  return 'textPrimary'
+}
+
+export const HealthDetails = ({ health }: { health: Health }) => {
+  const theme = useTheme()
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      sx={{
+        padding: Spacing.md,
+      }}
+    >
+      <Box
+        display="flex"
+        flexDirection="column"
+        gap={2}
+        sx={{
+          padding: Spacing.md,
+          border: 2,
+          borderColor: (t) => t.design.Layer.Highlight.Fill,
+        }}
+      >
+        <Box display="grid" gridTemplateColumns="auto 1fr" alignItems="end" gap={5}>
+          <Metric
+            label={t`Health`}
+            value={Number(health?.value)}
+            loading={health?.loading}
+            valueOptions={{ unit: 'percentage', decimals: 2, color: getHealthValueColor(health?.value ?? 0, theme) }}
```

Comment:
If you inline `useTheme()` here you can use arrow-body notation and reduce indenting by a level :nerd_face: